controller_manager/include/controller_manager/controller_manager.h

```diff
@@ -224,9 +214,9 @@ class ControllerManager{
   /// Double-buffered controllers list
   std::vector<ControllerSpec> controllers_lists_[2];
   /// The index of the current controllers list
-  int current_controllers_list_;
+  int current_controllers_list_ = 0;
```

Comment:
Consider using brace-initialization here.
```suggestion
  int current_controllers_list_{0};
```